0001-RM-18433-MRP-Freeze-the-proposals.patch

```diff
@@ -0,0 +1,356 @@
+From c4fd345a62f2e29f92a553cb4a026129bf5dd5dd Mon Sep 17 00:00:00 2001
```

Comment:
Why you have added this file? Please remove this.Take care of this unnecessary changes